packages/gatsby-plugin-gatsby-cloud/src/components/buttons/IndicatorButton.js

```diff
@@ -32,15 +32,27 @@ export default function IndicatorButton({
       >
         <div
           data-testid={`${testId}-button`}
+          onClick={
+            hoverable
+              ? onClick
+              : () => {
+                  setShowTooltip(!showTooltip)
+                }
```

Comment:
Just a nit but not a huge fan of inlining functions like this, I think it makes it harder to read even if they're straightforward